src/scenes/ObservationsRouter/details/index.tsx

```diff
@@ -72,15 +72,17 @@ export default function ObservationDetails(props: ObservationDetailsProps): JSX.
   }, [query]);
 
   const observationsResults = useAppSelector((state) =>
-    searchObservations(
-      state,
-      plantingSiteId,
-      selectedOrganization,
-      defaultTimeZone.get().id,
-      searchProps.search,
-      searchProps.filtersProps?.filters?.zone?.values ?? [],
-      status
-    )
+    !selectedOrganization?.id
```

Comment:
You should be able to use `!selectedOrganization` here -- the ID is guaranteed to be valid if the organization object exists at all (that was the idea with my change in commit a19055154).